api/envoy/admin/v3/certs.proto

```diff
@@ -56,6 +56,9 @@ message CertificateDetails {
 
   // Indicates the time at which the certificate expires.
   google.protobuf.Timestamp expiration_time = 6;
+
+  // Seconds until expiration of the certificate's OCSP response.
+  uint64 seconds_until_ocsp_response_expiration = 7;
```

Comment:
Nit: should this be a `google.protobuf.Duration`?

Reply:
I was trying to stay consistent with the `days_until_expiration` field just above, but am happy to change it!

Reply:
@htuch from an API perspective, do you think prefer that this switches to google.protobuf.Duration (correctness) vs using uint64 (consistency by matching `days_until_expiration`)?

I'll go ahead and add an OCSP expiration timestamp.